### PR TITLE
[6.2.z][Cherry-pick] Added UI Content View test to automate BZ1317828

### PR DESCRIPTION
**Pull request by abalakh**
https://bugzilla.redhat.com/show_bug.cgi?id=1426687
```python
py.test tests/foreman/ui/test_contentview.py -k test_negative_readonly_user_add_remove_repo
============================= test session starts ==============================
platform darwin -- Python 2.7.13, pytest-2.9.2, py-1.4.32, pluggy-0.3.1
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: xdist-1.15.0, cov-2.3.1
collected 82 items
2017-02-24 17:35:21 - conftest - DEBUG - Found WONTFIX in decorated tests ['1269


6.2.z specific changes to backport the test
196', '1103157', '1156555', '1230902', '1245334', '1204686', '1267224', '1226425', '1217635', '1079482', '1214312']

2017-02-24 17:35:21 - conftest - DEBUG - Collected 82 test cases


tests/foreman/ui/test_contentview.py s

==== 81 tests deselected by '-ktest_negative_readonly_user_add_remove_repo' ====
================== 1 skipped, 81 deselected in 14.97 seconds ===================
```

### CODE REVIEW

File: tests/foreman/ui/test_contentview.py [hunk 1]
```diff
@@ -106,8 +106,13 @@ def setup_to_create_cv(self, repo_name=None, repo_url=None, repo_type=None,
                 reposet=rh_repo['reposet'],
                 releasever=rh_repo['releasever'],
             )
+        old_task_timeout = entity_mixins.TASK_TIMEOUT
+        # Update timeout to 10 minutes to finish sync
+        entity_mixins.TASK_TIMEOUT = 600
         # Sync repository
         entities.Repository(id=repo_id).sync()
+        entity_mixins.TASK_TIMEOUT = old_task_timeout
```
Comment (oshtaier) [inline tests/foreman/ui/test_contentview.py]: `try, finally`?

**Comment by oshtaier** [comment]
ACK

File: tests/foreman/ui/test_contentview.py (hunk 1 above)
Comment (abalakh) [inline tests/foreman/ui/test_contentview.py]: I know, these lines were picked up directly from master with no changes. Do you want me to fix in 6.2.z only or in separate pr to master then cherry-pick to 6.2.z?

**Comment by renzon** [comment]
I am reviewing PRs but it seem this one is only waiting for @oshtaier

**Comment by oshtaier** [comment]
ACK

**Pull request merged**